src/program/render.rs

```diff
@@ -366,6 +371,7 @@ fn normal_help_text() -> Vec<&'static str> {
         "+------------+----------------------------+-----------------------------------------------------+",
         "| Key 1      | Key 2                      | Action                                              |",
         "+------------+----------------------------+-----------------------------------------------------+",
+        "| 0-9        |                            | Perform the below actions in bulk                   |",
```

Comment:
This is a huge nitpick, but maybe include that it takes one of the below operations?

Reply:
True
In the Key 2 slot. 
Text: `Key of the action to perform`